tests/fxl-properties-001/META-INF/container.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" ?>

```

Comment:
Same as the first comment in https://github.com/w3c/epub-tests/pull/31/files — this doesn't match the proposed directory names.

I think you should:
1) Edit https://w3c.github.io/epub-specs/epub33/rs/#layout to add a specific anchor to *each* of the normative statements (per step 10 at https://github.com/w3c/epub-tests#step-by-step)

2) While you're at it, add a data-tests attribute there, pointing back to your new tests

3) Rename these tests to match the new normative statements, as explained at https://github.com/w3c/epub-tests#naming.

Note: a whole bunch of the *current* tests do not follow this scheme. That's because I haven't finished my renaming pass yet; my reorganization PR last week was blocked for months on me not getting to that. That's why I didn't submit for months — to avoid this confusion — but since I wanted to get that in, please ignore the names of the current tests for now and follow the documentation, or the cmt-* examples. Thanks!